webpages/db.php

```diff
@@ -0,0 +1,11 @@
+<?php
+$servername = $username = $passd = "";
```

Comment:
add this file to .gitignore file, and put dummy values in this file which the users can change to test on their systems. read more about gitignore from https://git-scm.com/docs/gitignore